application/views/template.php

```diff
@@ -1,17 +1,32 @@
-<!DOCTYPE html>
-<html lang="en">
-	<head>
-		<meta charset="utf-8">
-		<title>{pagetitle}</title>
+<?php
+if (!defined('APPPATH'))
+	exit('No direct script access allowed');
```

Comment:
I think this indent is a tab as well. you probably copied it over from elsewhere. Fix please.